web-api/src/dispatchers/ses/sendBulkTemplatedEmail.ts

```diff
@@ -15,77 +22,85 @@ import { backOff } from '../../../../shared/src/tools/helpers';
  *   myCustomVar1: 'undefined',
  *   myCustomVar2: 'undefined'
  * }
- *
- * @param {object} providers the providers object
- * @param {object} providers.applicationContext application context
- * @param {object} providers.defaultTemplateData default values correlated with templateData matching the format described above
- * @param {Array} providers.destinations array of destinations matching the format described above
- * @param {string} providers.templateName name of the SES template
- * @returns {void}
  */
 export const sendBulkTemplatedEmail = async ({
   applicationContext,
   defaultTemplateData,
   destinations,
   templateName,
-}) => {
+}: {
+  applicationContext: ServerApplicationContext;
+  defaultTemplateData: { [key: string]: any };
+  destinations: { email: string; templateData: { [key: string]: any } }[];
+  templateName: string;
+}): Promise<void> => {
   try {
+    const params: SendBulkTemplatedEmailCommandInput = {
+      DefaultTemplateData: JSON.stringify(defaultTemplateData),
+      Destinations: destinations.map(destination => ({
+        Destination: {
+          ToAddresses: [destination.email],
+        },
+        ReplacementTemplateData: JSON.stringify(destination.templateData),
+      })),
+      ReturnPath:
+        process.env.BOUNCED_EMAIL_RECIPIENT ||
```

Comment:
It looks like the convention for `environment` is for props to have a default.  So I think I'll play it safe and update it here since the default is obvious but leave the other process.env refs alone where I'm less sure (e.g. `getEmailClient`)